src/web/utils.ts

```diff
@@ -227,3 +232,43 @@ function spherical2tilt(altitudeAngle: number, azimuthAngle: number) {
 
   return { tiltX, tiltY };
 }
+
+const SVGElements = [
+  'Circle',
+  'ClipPath',
+  'Ellipse',
+  'ForeignObject',
+  'G',
+  'Image',
+  'Line',
+  'Marker',
+  'Mask',
+  'Path',
+  'Pattern',
+  'Polygon',
+  'Polyline',
+  'Rect',
+  'Svg',
+  'Symbol',
+  'TSpan',
+  'Text',
+  'TextPath',
+  'Use',
+];
+
+// This function helps us determine whether given node is SVGElement or not. In our implementation of
+// findNodeHandle, we can encounter such element in 2 forms - SVG tag or ref to SVG Element. Since Gesture Handler
+// does not depend on SVG, we use our simplified SVGRef type that has `elementRef` field. This is something that is present
+// in actual SVG ref object.
+//
+// In order to make sure that node passed into this function is in fact SVG element, first we check if its constructor name
+// corresponds to one of the possible SVG elements. Then we also check if `elementRef` field exists.
+// By doing both steps we decrease probability of detecting situations where, for example, user makes custom `Circle` and
+// we treat it as SVG.
+export function isSVGElement(viewRef: SVGRef | GestureHandlerRef) {
+  const className = Object.getPrototypeOf(viewRef).constructor.name;
```

Comment:
Maybe we should rename it to avoid confusion with the React `className` prop. https://react.dev/learn#adding-styles

Reply:
Changed to `componentClassName` in [0722672](https://github.com/software-mansion/react-native-gesture-handler/pull/3197/commits/0722672fafccfc11363e210126d38e95d2ad2fcd), let me know what do you think 😅